res/skins/QMLDemo/Mixxx/Controls/Knob.qml

```diff
@@ -14,8 +20,18 @@ Item {
     property real wheelStepSize: (root.max - root.min) / 10
     property real angle: 130
     property bool arc: false
+    property int arcStart: Knob.Center
     property real arcRadius: width / 2
-    property real arcStart: (max - min) / 2
+    property real arcStartValue: {
```

Comment:
```suggestion
    readonly property real arcStartValue: {
```
?

Otherwise there can be contradicting states which is confusing.

Reply:
Well I purposefully wanted to let it read-write so if none of the ArcStart Enum values suits you. You can still specify the it yourself. The ArcStartValue takes precedence over ArcStart. There won't be an invalid state really (at least how the component is currently implemented). The only downside I see with this approach is that it does make the component a bit tricker to understand and change...

Reply:
Should I just add a comment that explains the relationship?

Reply:
Yes, let's make it readonly and revisit it later if we need to.